go/libraries/doltcore/schema/schema_impl.go

```diff
@@ -417,6 +417,10 @@ func (si *schemaImpl) GetKeyDescriptor() val.TupleDesc {
 	useCollations := false // We only use collations if a string exists
 	var collations []sql.CollationID
 	_ = si.GetPKCols().Iter(func(tag uint64, col Column) (stop bool, err error) {
+		if col.Virtual {
```

Comment:
What does it mean for a Virtual column to be in PK? How is this going to interact with logic which expects clustered indexes or whatever? Is this really valid?